kumascript/macros/LearnSidebar.ejs

```diff
@@ -638,6 +640,20 @@ const sections = [
       }
     ]
   },
+  {
+    name: "Games_Developing_for_web",
+    link: "../Games",
+    subsections: [
+      {
+        name: "Guides_and_tutorials",
+        pages: [
+          "../Games/Introduction",
+          "../Games/Tutorials",
+          "../Games/Publishing_games",
```

Comment:
@caugner do you know if there's a different way to handle relative paths like this, or is `../` okay to use here?

Reply:
It was the only way I could get out of the Learn section. It work, but there might be a better way :)

Reply:
I guess the best would be to move the Games section inside the Learn section, but otherwise I'm glad that the `../` syntax works. It makes it obvious that we're including pages from a different section.

Reply:
>move the Games section inside the Learn section

That's a good idea. We should keep this in mind for Learn area refresh when we get to it.